quinn-proto/src/config.rs

```diff
@@ -428,12 +430,13 @@ where
     S: crypto::Session,
 {
     /// Create a default config with a particular `token_key`
-    pub fn new(token_key: S::HmacKey) -> Self {
+    pub fn new(token_key: S::HmacKey, master_key: Vec<u8>) -> Self {
```

Comment:
I think we'd only want to ask for one key here. Presumably the HMAC key can be derived from the master key if necessary.